doc/Alerting/Rules.md

```diff
@@ -95,7 +95,7 @@ SELECT *,AVG(processors.processor_usage) as cpu_avg FROM devices,processors WHER
 ## Procedure
 
 You can associate a rule to a procedure by giving the URL of the
-procedure when creating the rule. Only links like "http://" are
+procedure when creating the rule. Only links like "<http://>" are
```

Comment:
How does that display to the user? If it shows `<http://>` then that's wrong and it should be just `http://`. Same for the other changes like this.

Reply:
That's how you create a link in markdown

Reply:
Yeah, but we don't want that to be a clickable link.

Reply:
It's because it wont lint else, and it's not possible to ignore it.  
We can switch linter to another one that allows to ignore "single offenders".  
I will look into it.